app/Http/Controllers/Bible/TextControllerV2.php

```diff
@@ -12,7 +12,6 @@
 use App\Traits\AccessControlAPI;
 use App\Traits\CheckProjectMembership;
 use App\Traits\CallsBucketsTrait;
-use App\Transformers\FontsTransformer;
```

Comment:
It has been removed because the FontsTransformer is not used in this controller.